gridstatus/pjm.py

```diff
@@ -329,6 +420,24 @@ def get_lmp(
             verbose=verbose,
         )
 
+        data = self._finalize_pjm_api_lmp_data(
+            data,
+            location_type,
+            locations,
+            market,
+            market_type,
+        )
+
+        return data
+
+    def _finalize_pjm_api_lmp_data(
```

Comment:
personal preference, but since this function isn't used anywhere else, I would suggest just keeping everything in the monolithic function and leaving a comment that is it about finalization